### ISSUE TITLE
Import hierarchy scale issue

### ISSUE THREAD

**Issue by yaustar** [playcanvas/editor#455]
Project: https://playcanvas.com/project/812065/overview/import-hierarchy-issue

Using import hierarchy on this FBX creates a template that has it's render assets set to 100 and is 100 times large than when imported as a model asset

Note: When first adding the hierarchy template to the scene, the mesh did not render and required a refresh of the browser.

See video below

Not 100% sure if this is an engine or editor issue

https://user-images.githubusercontent.com/16639049/125265556-dc43d100-e2fc-11eb-9037-e421a895aa74.mp4

**Comment by mvaligursky** [comment]
The scaling is likely related to the fact that the fbx is exported in cm units. @slimbuck - do you know much about how this is handled? Do we automatically convert to meters, or do we expect files to be in meters already?

**Comment by mvaligursky** [comment]
| Note: When first adding the hierarchy template to the scene, the mesh did not render and required a refresh of the browser.


Fix for this part: https://github.com/playcanvas/engine/pull/3328

**Comment by slimbuck** [comment]
> The scaling is likely related to the fact that the fbx is exported in cm units. @slimbuck - do you know much about how this is handled? Do we automatically convert to meters, or do we expect files to be in meters already?

Yes correct, backend converts models to meters and scales the parent accordingly.

**Comment by mvaligursky** [comment]
@ellthompson - can you please investigate the part where the mesh does not animate. Make sure to get latest engine to have the fix I mention above.

**Comment by yaustar** [comment]
Should it be in a different 'pose' too? (the video above has no animation)

**Comment by willeastcott** [comment]
I imported the FBX as both template asset (left) and model asset (right):

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/f7f2730e-ed81-4bbd-b108-9e83417a2b0a" />

They look identical and render correctly straight away on import.

Closing.
